### PR TITLE
[14491] Label changes in eMediplan added l10n

### PR DESCRIPTION
**Pull request by jamda**

### CODE REVIEW

File: bundles/at.medevit.elexis.emediplan.ui/OSGI-INF/l10n/plugin.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+#Properties file for at.medevit.elexis.emediplan.ui
+
+Bundle-Name = eMediplan Ui
+
+Bundle-Vendor = medevit.at
+
```
Comment (col-panic) [inline bundles/at.medevit.elexis.emediplan.ui/OSGI-INF/l10n/plugin.properties]: Do not translate these values (Bundle-Name and vendor)

Comment (jamda) [reply bundles/at.medevit.elexis.emediplan.ui/OSGI-INF/l10n/plugin.properties]: Ok. i think it's safer too!

File: bundles/ch.elexis.icpc/src/ch/elexis/icpc/views/EpisodesView.java [hunk 2]
```diff
@@ -199,7 +198,7 @@ public void run(){
 		
 		editEpisodeAction = new Action("Problem bearbeiten") {
 			{
-				setToolTipText("Titel des Problems ändern");
+				setToolTipText("Problem bearbeiten");
 				setImageDescriptor(Images.IMG_EDIT.getImageDescriptor());
```
Comment (col-panic) [inline bundles/ch.elexis.icpc/src/ch/elexis/icpc/views/EpisodesView.java]: Fehlt hier nicht eine Übersetzung?

Comment (ngiger) [reply bundles/ch.elexis.icpc/src/ch/elexis/icpc/views/EpisodesView.java]: As the ICPC contains at least 250 non externalized strings, I propose that Txomin shall do this work later and that Marco just merges the text change as requested by Matej

**Pull request merged**